reference/vpp/host-network.md

```diff
@@ -0,0 +1,94 @@
+${SERVICE_CIDR} via 192.168.0.254 dev enp216s0f1 proto static mtu 1440
+${POD_CIDR} via 192.168.0.254 dev enp216s0f1 proto static mtu 1440
+192.168.0.0/24 dev enp216s0f1 proto kernel scope link src 192.168.0.254
+````
+The nex `tap` interface is also configured with routes towards the Kubernetes service CIDR and the pod CIDR, so that the linux host can reach the workloads through VPP. These routes use a reduced MTU to accomodate for encapsulations.
+
+* In VPP you will find it with the name `tap0`
+  * It is configured unnumbered, as a child of the primary interface `avf-0/d8/a/0`
+````
+vpp# sh int addr
+tun0 (up):
+  unnumbered, use avf-0/d8/a/0
+  L3 192.168.0.1/24
+````
+* It is also registered as the default punt path
+  * This means that all the traffic that would be dropped by VPP (which includes the traffic to the VPP address that is not handled by VPP itself, but not the tunnel traffic which is decapsulated / decrypted by VPP) will be passed to this interface, so the linux host will receive and process it
+````
+vpp# sh ip punt redirect
+ rx local0 via:
+   path-list:[31] locks:1 flags:no-uRPF, uPRF-list:24 len:1 itfs:[2, ]
+    path:[41] pl-index:31 ip4 weight=1 pref=0 attached-nexthop:  oper-flags:resolved,
+      169.254.0.1 tap0
+    [@0]: ipv4 via 169.254.0.1 tap0: mtu:1500 next:6 flags:[features ] 52540040468e02ca11c0fd100800
+ forwarding
+  [@1]: ipv4 via 169.254.0.1 tap0: mtu:1500 next:6 flags:[features ] 52540040468e02ca11c0fd100800
+# For the v6 configuration, use
+vpp# sh ip6 punt redirect
+````
+
+### Packet flow - incoming packet on the uplink
+
+A packet destined to the host arrives on the uplink, let's say it has `src=192.168.0.2,dst=192.168.0.1`
+* VPP receives it, sees that the destination address is the one configured on its interface
+* As it doesn't have specific handling configured for this packet, it looks up the punt path, and sends it into `tap0`
+* Linux receives it on the tap interface which is configured with `192.168.0.1/32` and so processes it normally
+
+The reply is now emitted by the host with `src=192.168.0.1,dst=192.168.0.2`
+* Linux looks up the route for `192.168.0.0/24`, forwards it into the tap interface
+* VPP receives it on the interface `tap0`
+* It looks up `192.168.0.2` in the fib, finds a route on the uplink interface (configured with `192.168.0.1/24`)
+* The packet is sent on the uplink
+
+### Packet flow - pod talking to the api-server
+
+Let's say the api server lives on `Node A`
+
+If a pod on `Node B` wants to talk to the api-server on `Node A`, the packet flow will be the same as described above.
+* In `Node B`, standard routing happens (`sh ip fib <someip>` will give details on this node)
+* When reaching `Node A` we're in the same situation as previously
+
+
+If a pod on `Node A` wants to talk to the api-server on `Node A`, let's say the packet is `src=10.0.0.1,dst=192.168.0.1` with `10.0.0.1` being the pod address. 
```

Comment:
If a pod on `Node A` wants to talk to the api-server on `Node A`, let's say the packet is `src=10.0.0.1,dst=192.168.0.1` with `10.0.0.1` being the pod address, then: